packages/waas/src/auth.ts

```diff
@@ -249,50 +255,78 @@ export class SequenceWaaS {
     }
   }
 
-  async initiateEmailAuth(email: string) {
-    const intent = await this.waas.initiateEmailAuth(email)
+  async initAuth(identity?: Identity): Promise<Challenge> {
+    if (!identity) {
+      return this.initiateGuestAuth()
+    }
+
+    if ('idToken' in identity) {
+      return this.initiateIdTokenAuth(identity.idToken)
+    } else if ('email' in identity) {
+      return this.initiateEmailAuth(identity.email)
+    } else if ('playFabTitleId' in identity) {
+      return this.initiatePlayFabAuth(identity.playFabTitleId, identity.playFabSessionTicket)
+    }
+
+    throw new Error('invalid identity')
+  }
+
+  private async initiateGuestAuth() {
```

Comment:
I think we should standard on `init[METHOD]Auth` for naming. It's nicer to keep it the same across the public method and internals.